configure.ac

```diff
@@ -1616,6 +1588,106 @@ else
     AC_DEFINE([CHIP_WITH_MBEDTLS], [1], [Define to 1 to build CHIP with mbedTLS])
 fi
 
+#
+# OpenThread
+#
+
+NL_WITH_OPTIONAL_INTERNAL_PACKAGE(
```

Comment:
This is all highly nRF5-specific. How can or does this work with ESP32? EFR32? Etc.?

Reply:
esp32 doesn't have Thread radio. I think each platform has to provide customized build options like nrf5. For example, some platform may not support hardware acceleration of encryption, so the mbedtls-config differs. Some has very restricted resources that they only support mtd.

Reply:
I can see that lines 1016 and 1017 are conditionalized by the `nrf5` switch case. However, re-reading this, I missed lines 1607 and 1608 which are also conditionalized by the `nrf5` switch case.

It makes sense now.